acme/challenge.go

```diff
@@ -466,6 +468,8 @@ func wireDPOP01Validate(ctx context.Context, ch *Challenge, db DB, jwk *jose.JSO
 		return WrapErrorISE(err, "error unmarshalling challenge data")
 	}
 
+	issuer := dpopOptions.DpopTarget
```

Comment:
There should be a `GetDPOPTarget()` function in order to avoid null-pointer panics.